### PR TITLE
Allow option to disable the preface in spec reporter

### PR DESCRIPTION
**Pull request by zmilonas**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by zmilonas** [comment]
Thanks for taking the time to check @christian-bromann. I did have one typo. You could have gotten this.

I am testing by running 
```./packages/wdio-cli/bin/wdio.js ./e2e/wdio/wdio.conf.js```

With `['spec', { showPreface: false }]` in the configuration as the reporter and there is no more preface.

Can you tell me what steps you're taking to arrive at this obviously not intentional output?

**Comment by christian-bromann** [comment]
@zmilonas I did not used your branch. When I run tests with the latest WebdriverIO version using Multiremote and the spec reporter, I receive this output:
```
 "spec" Reporter:
------------------------------------------------------------------
[MultiremoteBrowser #0-0] Running: MultiremoteBrowser on Chrome, Chrome
[MultiremoteBrowser #0-0]
[MultiremoteBrowser #0-0] » /multiremote/mocha.test.js
[MultiremoteBrowser #0-0] multiremote example
[MultiremoteBrowser #0-0]    ✓ should open chat application
[MultiremoteBrowser #0-0]    ✓ should login the browser A
[MultiremoteBrowser #0-0]    ✓ should login the browser B
[MultiremoteBrowser #0-0]    ✓ should post something in browserA
[MultiremoteBrowser #0-0]    ✓ should read the message in browserB
[MultiremoteBrowser #0-0]
[MultiremoteBrowser #0-0] 5 passing (6.9s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:09
```
I don't think this option is needed, given that I don't see any `unknown`. Can you confirm? Please test with the latest WebdriverIO version.

**Comment by zmilonas** [comment]
Hey @christian-bromann thank you so much for taking the time to review this again.

I might have not been clear engouh in my first entry. 

So what I'm looking for is to get more "regular" output with the option `{ showPreface: false }` like so:

```
"spec" Reporter:
------------------------------------------------------------------
 Running: MultiremoteBrowser on Chrome, Chrome

 » /multiremote/mocha.test.js
 multiremote example
    ✓ should open chat application
    ✓ should login the browser A
    ✓ should login the browser B
    ✓ should post something in browserA
    ✓ should read the message in browserB

 5 passing (6.9s)
````

File: packages/wdio-spec-reporter/README.md [hunk 1]
```diff
@@ -113,3 +113,18 @@ Default: `false`
   },
 ]
 ```
+
+### showPreface
+Set to false to disable `[ MutliRemoteBrowser (unknown) #0-0 ]` preface in the reports.
```
Comment (christian-bromann) [inline packages/wdio-spec-reporter/README.md:118]:
```suggestion
Set to false to disable `[ MutliRemoteBrowser #0-0 ]` preface in the reports.
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]
It would be great if we could add a unit test to ensure this works as expected. Thanks!

**Comment by christian-bromann** [comment]
@zmilonas any updates on the unit tests? Happy to support if something is unclear.

**Comment by zmilonas** [comment]
Hey @christian-bromann thanks for the reminder. I definitely will try to do that, just haven't had the time recently because of being away. I'll reach out if I need support.

**Comment by zmilonas** [comment]
@christian-bromann I added a snapshot tests, you can look again

**Review by christian-bromann — APPROVED** [review]
Awesome, thanks 👍

**Pull request merged**